### ISSUE TITLE
Code from importScripts() is not recognized in webextension service worker.

### ISSUE THREAD

**Issue by vanowm** [eslint/eslint#17650]
### Environment

Node version: v19.8.1
npm version: v9.6.4
Local ESLint version: v8.51.0 (Currently used)
Global ESLint version: v8.47.0
Operating System: win32 10.0.19045

### What parser are you using?

Default (Espree)

### What did you do?

<details>
<summary>Configuration</summary>

`.eslintrc.json`
```json
  "env": {
      "es6": true,
      "serviceworker": true,
      "worker": true,
      "webextensions": true
  },
  "rules": {
      "no-undef": "error"
  }
```

`debug.js`
```js
const debug = console.bind(console);
```
</details>

`service_worker.js`
```js
importScripts("debug.js")
debug.log("blah");
```

### What did you expect to happen?

Pass validation.

### What actually happened?

Error:
`'debug' is not defined. eslint(no-undef)`

### Link to Minimal Reproducible Example

https://eslint.org/play/#eyJ0ZXh0IjoiaW1wb3J0U2NyaXB0cyhcImRlYnVnLmpzXCIpXG5kZWJ1Zy5sb2coXCJibGFoXCIpO1xuIiwib3B0aW9ucyI6eyJlbnYiOnsic2VydmljZXdvcmtlciI6dHJ1ZX0sInJ1bGVzIjp7Im5vLXVuZGVmIjpbImVycm9yIl19LCJwYXJzZXJPcHRpb25zIjp7ImVjbWFGZWF0dXJlcyI6e30sImVjbWFWZXJzaW9uIjoibGF0ZXN0Iiwic291cmNlVHlwZSI6InNjcmlwdCJ9fX0=

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by Rec0iL99** [comment]
Hi @vanowm, thanks for the issue. Can you please refer #16904 and see if it helps? Thanks.

**Comment by vanowm** [comment]
Is there a way make eslint track global variables without creating a list of global variables manually?

**Comment by nzakas** [comment]
@vanowm No, sorry. If you create `debug`, then you'll need to define it yourself:

https://eslint.org/docs/latest/use/configure/language-options#specifying-globals
